index.html

```diff
@@ -18,27 +17,35 @@
   </head>
   <body>
     <div class="container">
-      <div class="introduction-header row">
-        <h1 class="header col s12"></h1>
-        <p class="slogan col s12"></p>
+      <div id="logo" class="introduction row">
+        <div class="col s8 offset-s2">
+          <h3 class="app-header center-align"></h3>
```

Comment:
Do you need the word app in these class names? Is header the best name, is it actually the site name?